astronomer/providers/amazon/aws/sensors/batch.py

```diff
@@ -42,7 +51,7 @@ def execute(self, context: Context) -> None:
                 job_id=self.job_id,
                 aws_conn_id=self.aws_conn_id,
                 region_name=self.region_name,
-                poll_interval=self.poll_interval,
+                poll_interval=self.poke_interval,
```

Comment:
```suggestion
                poke_interval=self.poke_interval,
```
We should rename all internal occurrences of this variable and accordingly also rename the variable in the triggers.

Reply:
@pankajkoti Yeah, we can change the internal occurrences for sensor and trigger, will raise separate PR for the operator